tests/integration/Mvc/Model/BinderCest.php

```diff
@@ -0,0 +1,500 @@
+use Phalcon\Mvc\Model\MetaData\Memory;
+use Phalcon\Test\Models\People;
+use Phalcon\Test\Models\Robots;
+
+/**
+ * \Phalcon\Test\Integration\Mvc\Model\BindingCest
+ * Tests the Phalcon\Mvc\Application component
+ *
+ * @copyright (c) 2011-2016 Phalcon Team
+ * @link          http://www.phalconphp.com
+ * @author        Andres Gutierrez <andres@phalconphp.com>
+ * @author        Serghei Iakovlev <serghei@phalconphp.com>
+ * @author        Wojciech Ślawski <jurigag@gmail.com>
+ * @package       Phalcon\Test\Integration\Mvc\Model
+ *
+ * The contents of this file are subject to the New BSD License that is
+ * bundled with this package in the file docs/LICENSE.txt
+ *
+ * If you did not receive a copy of the license and are unable to obtain it
+ * through the world-wide-web, please send an email to license@phalconphp.com
+ * so that we can send you a copy immediately.
+ */
+class BinderCest
+{
+    /**
+     * @var Apc
+     */
+    private $cache;
+
+    /**
+     * @var Binder
+     */
+    private $modelBinder;
+
+    /**
+     * @var Manager
+     */
+    private $modelsManager;
+
+    /**
+     * Executed before each test
+     *
+     * @param IntegrationTester $I
+     */
+    public function _before(IntegrationTester $I)
+    {
+        Di::setDefault($I->getApplication()->getDI());
+
+        $this->cache = new Apc(new Data(['lifetime' => 20]));
+        $this->modelBinder = new Binder($this->cache);
+
+        $this->modelsManager = $I->getApplication()->getDI()->getShared('modelsManager');
+
+        $I->haveServiceInDi('modelsMetadata', function () {
+            return new Memory;
+        }, true);
+    }
+
+    /**
+     * Tests dispatcher and model binder
+     *
+     * @author Wojciech Ślawski <jurigag@gmail.com>
+     * @since  2016-10-20
+     *
+     * @param IntegrationTester $I
+     */
+    public function testDispatcher(IntegrationTester $I)
+    {
+        $dispatcher = new Dispatcher;
+        $dispatcher->setModelBinder($this->modelBinder);
+        $dispatcher->setDI(Di::getDefault());
+
+        $I->assertInstanceOf('Phalcon\Di', $dispatcher->getDI());
+        $I->haveServiceInDi('dispatcher', $dispatcher);
+
+        $people = People::findFirst();
+
+        $dispatcher->setControllerName('test10');
+        $dispatcher->setActionName('view');
+        $dispatcher->setParams(['people' => $people->cedula]);
+
+        $dispatcher->dispatch();
+
+        $returnedValue = $dispatcher->getReturnedValue();
+        $I->assertInstanceOf('Phalcon\Test\Models\People', $returnedValue);
+        $I->assertEquals($people->cedula, $returnedValue->cedula);
+        $I->assertEquals([
+            'people' => 'Phalcon\\Test\\Models\\People',
+        ], $this->cache->get('_PHMB_Test10Controller_viewAction'));
+
+        $robot = Robots::findFirst();
+
+        $dispatcher->setActionName('multiple');
+        $dispatcher->setParams(['people' => $people->cedula, 'robots' => $robot->id]);
+
+        $dispatcher->dispatch();
```

Comment:
this is the second time you perform action. It is a good idea to make test to have 3 steps: arrange, act, assert. You already did assertion in this test, so this can be made as another test.

This is important because smaller tests are easier to read, understand, and fix. If setup part of those tests are similar you can move it to separate method and call it using `@before` annotation